include/eve/module/core/regular/impl/abs.hpp

```diff
@@ -36,12 +36,18 @@ abs_(EVE_SUPPORTS(cpu_), T const& a) noexcept
 
 // -----------------------------------------------------------------------------------------------
 // Masked case
-template<conditional_expr C, value U>
+template<conditional_expr C, scalar_value U>
 EVE_FORCEINLINE auto
 abs_(EVE_SUPPORTS(cpu_), C const& cond, U const& t) noexcept
 {
   return mask_op(cond, eve::abs, t);
 }
+template<conditional_expr C, scalar_value U, typename N>
+EVE_FORCEINLINE auto
+abs_(EVE_SUPPORTS(cpu_), C const& cond, wide<U,N> const& t) noexcept
+{
+  return mask_op(cond, eve::abs, t);
+}
```

Comment:
Why do you need this one now?

Reply:
I thought I was clever but I was not :D
Fixed

Reply:
It was indeed required then the other simplification made it uselesss. 
Removed